test/v6/lossClaims/delete/DeleteLossClaimServiceSpec.scala

```diff
@@ -79,6 +80,7 @@ class DeleteLossClaimServiceSpec extends ServiceSpec {
       val errors: Seq[(String, MtdError)] = List(
         "INVALID_TAXABLE_ENTITY_ID" -> NinoFormatError,
         "INVALID_CLAIM_ID"          -> ClaimIdFormatError,
+        "INVALID_TAX_YEAR"          -> TaxYearFormatError,
```

Comment:
This should be mapped to [TaxYearClaimedForFormatError](https://github.com/hmrc/individual-losses-api/blob/main/app/common/errors/LossesApiCommonErrors.scala#L29), thanks